### ISSUE TITLE
Uncaught Error: Cannot find module 'babel-runtime/helpers/get' from 'semantic-ui-react/dist/commonjs/modules/Dropdown/Dropdown.js'

### ISSUE THREAD

**Issue by amshim** [Semantic-Org/Semantic-UI-React#1260]
Semantic-UI-React Version: ^0.64.5 (Latest)

I get the following error when trying to import "semantic-ui-react" into any of my React components. I have tried installing babel-runtime to my dev packages, but still no luck. I am not even trying to use dropdown menus.

https://i.gyazo.com/34702d671ae499a2e419d62af995bfd4.png

**Comment by layershifter** [comment]
@amshim Try to remove `node_modules` folder and then `npm i`, seems you have same issue like in #1229.

**Comment by levithomason** [comment]
Hm, though this is likely true, we should actually have `babel-runtime` as a `peerDependency` I believe.  Since the commonjs modules need to require it.

**Comment by amshim** [comment]
> Try to remove node_modules folder and then npm i, seems you have same issue like in #1229.

I tried this and it does not resolve the issue. Error still comes up.

**Comment by levithomason** [comment]
@amshim please try `npm i babel-runtime` then run a build.  Our commonjs components are transformed to require modules from `babel-runtime`.  If you do not have it, then it will fail.

If that ^ works for you, probably will, then we just need to add this package as a peerDependency to our package.

**Comment by JulianIsrael** [comment]
having same issue importing semantic-ui-react using jspm

**Comment by levithomason** [comment]
Could someone please first install `babel-runtime` then try again and let me know what happens?  Almost certain this is the issue but I need some conformation before I ship a fix.

**Comment by amshim** [comment]
> Could someone please first install babel-runtime then try again and let me know what happens? Almost certain this is the issue but I need some conformation before I ship a fix.

I tried this and the problem still persisted.

**Comment by JulianIsrael** [comment]
the problem still there.

**Comment by levithomason** [comment]
Thanks much guys! In that case, anyone able to create a minimal repo that reproduces this error?  We'll have to dig in a bit more it appears.

**Comment by JulianIsrael** [comment]
> git clone https://github.com/JulianIsrael/semantic-ui-jspm-issue.git
> npm run init

when it finishes, it will open a browser and see the console error with the import. Versions before the 0.64.0 release works.

**Comment by layershifter** [comment]
Okay, I made it working. I've added `bs-config.json`:

```json
{
  "watchOptions": {
    "ignored": ["node_modules", "jspm_packages" ]
  }
}
```

Then `jspm install babel-runtime` and `npm run start`, all works. @JulianIsrael can you confirm?

----

@levithomason Seems we need add `babel-runtime` to dependencies.

**Comment by scholtzm** [comment]
Possibly related?

https://github.com/facebookincubator/create-react-app/issues/1421

In my case, the issue just "went away". Not sure where the actual culprit was hidden. 🤔

**Comment by tslater** [comment]
We get this issue intermittently, only in our CI setup, as well. It's really strange why it shows up there and not locally.

**Comment by levithomason** [comment]
OK folks, PR up.  Thanks for all the extra work trialing things here.  This should most certainly fix all the issues noted here.  Will try to get  release out tonight.

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.64.8`.

**Comment by tslater** [comment]
Awesome work...like always! Thanks @levithomason!
